src/module/sheet/helpers.ts

```diff
@@ -32,6 +32,23 @@ function createSheetTags(options: Record<string, string>, selections: SheetSelec
     return createSheetOptions(options, selections, { selected: true });
 }
 
+/**
+ * Process tagify elements in a form, converting their data into something the pf2e system can handle.
+ * This method is meant to be called in _getSubmitData().
+ */
+function processTagifyInSubmitData(form: HTMLFormElement, data: Record<string, unknown>) {
+    // Tagify has a convention (used in their codebase as well) where it prepends the input element
+    const tagifyInputElements = form.querySelectorAll<HTMLInputElement>("tags.tagify ~ input");
+    for (const inputEl of tagifyInputElements.values()) {
+        const path = inputEl.name;
+        const inputValue = data[path];
+        const selections = inputValue && typeof inputValue === "string" ? JSON.parse(inputValue) : inputValue;
+        if (Array.isArray(selections)) {
+            data[path] = selections.map((w: { id?: string; value?: string }) => w.id ?? w.value);
+        }
+    }
+}
+
```

Comment:
Can this be used in homebrew settings as well?

Reply:
Not sure, I can't find any tagify code other than the onSubmit override. I think homebrew settings use string list tagify's, which do not need any post-processing.

Reply:
It doesn't

Reply:
When I was taking a crack at a campaign feats editor before I underwent burnout, I did add this to the settings menu. It'll probably come back around then.

The one other place this will come in handy is journals.